cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/Context.java

```diff
@@ -215,39 +222,85 @@ public void anotherLineConversion(ACLineSegmentConversion c) {
     }
 
     public void endLinesConversion() {
-        String enough = countLinesWithSvPowerFlowsAtEnds < countLines ? "FEW" : "ENOUGH";
-        LOG.info("{} lines with SvPowerFlow values at ends: {} / {}",
-                enough,
-                countLinesWithSvPowerFlowsAtEnds,
-                countLines);
+        if (LOG.isInfoEnabled()) {
+            String enough = countLinesWithSvPowerFlowsAtEnds < countLines ? "FEW" : "ENOUGH";
+            LOG.info("{} lines with SvPowerFlow values at ends: {} / {}",
+                    enough,
+                    countLinesWithSvPowerFlowsAtEnds,
+                    countLines);
+        }
     }
 
     public void invalid(String what, String reason) {
-        LOG.warn("Invalid {}. Reason: {}", what, reason);
+        if (LOG.isWarnEnabled()) {
+            LOG.warn(INVALID_REASON, what, reason);
```

Comment:
I think adding a check for log level enabled before a call with string constants does not offer performance improvement, the log library will do the same check in the implementation (https://stackoverflow.com/questions/8788096/log-info-using-log-isinfoenabled). Personally I prefer removing the check for such one-line calls, to improve readability.

Reply:
I see your point, but then it's like for the partial use of Suppliers with @mathbagu : what's the policy ? Should we say "prefer doing stuff that way in general" or "work on a case by case basis" ?

Reply:
Looking to the code of logback, the log methods use a chain of filter to define to decide if a LogEvent has to be created. There are two different implementation of LogFilter, and in that case, the TurboFilter are used. These filters have to be really fast. So I think, we can follow Luma's advice.